### PR TITLE
add moes curtain switch TS0601 _TZE200_nhyj64w2

### PR DESCRIPTION
**Pull request by 1subu**
Add support for Moes Cover/Curtain Switch TS0601 _TZE200_nhyj64w2

### CODE REVIEW

File: devices/moes.js [hunk 1]
```diff
@@ -325,4 +325,16 @@ module.exports = [
         exposes: [e.cover_position(), exposes.numeric('calibration_time', ea.ALL).withValueMin(0).withValueMax(100),
             exposes.enum('moving', ea.STATE, ['UP', 'STOP', 'DOWN']), exposes.binary('motor_reversal', ea.ALL, 'ON', 'OFF')],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_nhyj64w2'}],
+        zigbeeModel: ['TS0601'],
```
Comment (Koenkk) [inline devices/moes.js:330]: Please remove this line, fingerprint should be enough

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -8528,6 +8528,41 @@ const converters = {
...
+            switch (dp) {
+            case tuya.dataPoints.coverPosition: {
+                const invert = !tuya.isCoverInverted(meta.device.manufacturerName) ?
+                    !options.invert_cover : options.invert_cover;
+                const position = invert ? 100 - value : value;
+                result = {position: position};
+                break;
+            }
+            case tuya.dataPoints.state:
+                result = {state: {0: 'OPEN', 1: 'STOP', 2: 'CLOSE'}[value], running: {0: 'true', 1: 'false', 2: 'true'}[value]};
```
Comment (Koenkk) [inline converters/fromZigbee.js:8549]: Please make these values real booleans: `{0: 'true', 1: 'false', 2: 'true'}` -> `{0: true, 1: false, 2: true}`

**Comment by 1subu** [comment]
All changes done. Model is ZTS-EUR-C.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**